README.md

```diff
@@ -54,6 +54,30 @@ oc create secret generic apicast-configuration-url-secret \
 oc new-app -f https://raw.githubusercontent.com/3scale/apicast/master/openshift/apicast-template.yml
 ```
 
+### OKD (Openshift-origin 3.11)
+
+There are multiple approaches to setup 3scale APIcast on Openshift origin. The one with yaml file is the common approach to setup 3scale account. Another approach is the easiest one where you need to pull 3scale image on local docker repository, then create new application in OKD from docker image by passing required parameters. I have used this approach to setup 3scale APIcast on my all in one OKD cluster on AWS.
+
+- Pull the 3scale image to your local docker repository.
+
+docker pull registry.access.redhat.com/3scale-amp20/apicast-gateway:1.0
+
+In all cases, in order to setup 3scale you need to have 3scale account. Steps to create and configure 3scale account are given here (https://github.com/tnscorcoran/rhsso-3scale).
+
+- Once configured 3scale account, you need 'ACCESS_TOKEN' along with your 3scale account URL which is called `ADMIN_PORTAL_DOMAIN`, to create secret which in turn used by your application for authentication. Create secret as discussed in the above approach.
+
+oc create secret generic apicast-configuration-url-secret --from-literal=password=https://2040edXXXXXXXXX80d6@XXXXX-admin.3scale.net --type=kubernetes.io/basic-auth
+
+Using 'ACCESS_TOKEN' and 'ADMIN_PORTAL_DOMAIN', you can configure another environment variable 'THREESCALE_PORTAL_ENDPOINT'.
+
+THREESCALE_PORTAL_ENDPOINT=https://ACCESS_TOKEN@ADMIN_PORTAL_DOMAIN
+
+- In order to create application using local 3scale docker image, you need to pass environment variables
+
+oc new-app --docker-image=registry.access.redhat.com/3scale-amp20/apicast-gateway:1.0  --param CONFIGURATION_URL_SECRET=apicast-configuration-url-secret -e THREESCALE_PORTAL_ENDPOINT=https://2040edXXXXXXXXX80d6@XXXXX-admin.3scale.net
+
+This should setup 3scale APIcast on your all in one OKD cluster.
```

Comment:
Is there a visible way to the user to show that the configuration is complete?

Reply:
I used to check service hit count  in 3scale accout

Reply:
Perhaps you can include a verification step, so the user can confirm that the config is complete.